src/validation.module.ts

```diff
@@ -0,0 +1,10 @@
+import { Module } from '@nestjs/common';
+import { GenericValidator } from './domain/schema/generic.validator';
+import { JsonSchemaService } from './domain/schema/json-schema.service';
+import { ValidationErrorFactory } from './domain/schema/validation-error-factory';
+
+@Module({
+  providers: [JsonSchemaService, GenericValidator, ValidationErrorFactory],
+  exports: [JsonSchemaService, GenericValidator, ValidationErrorFactory],
+})
+export class ValidationModule {}
```

Comment:
This module can now be used in the `AppModule`. If you prefer to do it in a separate PR or here either way it's fine 👍